R/toml_serialization.R

```diff
@@ -41,16 +41,15 @@ to_toml <- function(...,
   names <- names2(args)
 
   # We disallow unnamed top-level atomic arguments
-  invalid <- which(map_lgl(args, ~ is_atomic(.x) && !is.null(.x)))
+  invalid <- which(!nzchar(names) & vapply(args, is.atomic, logical(1)))
```

Comment:
Again, why ditching `purrr` in favor of base R?

Reply:
This was preventative based on the above experience with `extract_meta()`.

Reply:
I'm ruling in favour of tidyverse here. If nothing else it's more concise.

Reply:
Resolved.